app/pb_kits/playbook/pb_date/_date.jsx

```diff
@@ -1,21 +1,59 @@
+/* @flow */
+
 import React from 'react';
-import PropTypes from "prop-types";
-
-const propTypes = {
-  className: PropTypes.string,
-  id: PropTypes.string
-};
-
-class Date extends React.Component {
-  render() {
-    return (
-      <div className="pb_date">
-        <span>DATE CONTENT</span>
-      </div>
-    )
-  }
+import DateTime from '../pb_kit/dateTime.js'
```

Comment:
Why do you need `DateTime`?

Reply:
I'm using DateTime to build an object for `dateTimeValue`. I tried deleting this line to see if it was required, and the date string wont display without it. Is there a better way?

Reply:
I believe that with the proposal above, we would not need `DateTime`, `moment.js` can handle all date manipulation we need.

Reply:
[I think we should avoid `moment` in PB](https://github.com/powerhome/playbook/pull/473#discussion_r354384331)

Reply:
Interesting. The reason we chose `moment` is because that [seemed to be the "norm"](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_kit/dateTime.js) in PB.

Is it the plan to migrate to `date-fns`? If so we can look into replacing the moment implementation with `date-fns`.

Reply:
nvm, just saw your comment below. @Annihilous wanna give that a shot?

Reply:
@thestephenmarshall have you guys looked into [dayjs](https://github.com/iamkun/dayjs)?